opennlp-tools/src/main/java/opennlp/tools/ml/model/DataIndexer.java

```diff
@@ -80,9 +81,14 @@
   /**
    * Sets parameters used during the data indexing.
    * @param trainParams
-   * @param reportMap
    */
-  public void init(Map<String, String> trainParams, Map<String, String> reportMap);
+  public void init(Map<String, String> trainParams);
```

Comment:
yes, but I didn't want to delete the method until there was some comment on the code.  I would like to remove all the Map<String,String> trainParams, and replace it with the TrainingParameters.